src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body,
+.header,
+.nav__list {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  background: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: #fff;
+  margin-bottom: 74px;
```

Comment:
for what?

![image](https://user-images.githubusercontent.com/70009205/202842104-d74f5613-44cc-42f5-8843-4e7c48701dd9.png)